Source/WebGPU/WebGPU/Queue.swift

```diff
@@ -23,7 +23,39 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+private let largeBufferSize = 32 * 1024 * 1024
 
-public func writeBuffer(queue: WebGPU.Queue, buffer: WebGPU.Buffer, offset: UInt64, data: SpanUInt8) {
-    queue.writeBuffer(buffer.buffer(), offset, data)
+public func writeBuffer(
+    queue: WebGPU.Queue, buffer: WebGPU.Buffer, bufferOffset: UInt64, data: SpanUInt8
+) {
+    queue.writeBuffer(mtlBuffer: buffer.buffer(), bufferOffset: bufferOffset, data: data)
+}
+
+extension WebGPU.Queue {
+    public func writeBuffer(mtlBuffer: any MTLBuffer, bufferOffset: UInt64, data: SpanUInt8) {
```

Comment:
can this use `some` instead of `any`?

Reply:
Good catch. Yes it can be `some`. Will do that in the next one.